test/phonon/helpers.jl

```diff
@@ -0,0 +1,162 @@
+using Random
+using ForwardDiff
+using LinearAlgebra
+
+# Helpers functions for tests.
+# TODO: Temporary, explanations too scarce. To be changed with proper phonon computations.
+
+# Convert back and forth between Vec3 and columnwise matrix
+fold(x)   = hcat(x...)
+unfold(x) = Vec3.(eachcol(x))
```

Comment:
Too generic names. This will lead to name clashes at some point. Maybe put all the phonon test helpers in a submodule or wrap all phonon tests in a common `@testset` ?